xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+
+ <sect1 xml:id="sec.ha.pmremote.usage">
+  <title>Usage Scenario</title>
+  <para>
+   The procedures in this document will lead to a minimal setup of a cluster
+   with the following properties:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <remark>toms 2017-03-23: Is this supported from 12 GA, SP1, or SP2?
+    According to aspiers, since SP1 at least</remark>
+    <para>A cluster node running &sls; 12 SP2. In this guide it is named
+     <systemitem class="domainname">&wsI;</systemitem> as its hostname.</para>
+   </listitem>
+   <listitem>
+    <para>Failover of resources from one node to the other if the active host
+     breaks down (active/passive setup).</para>
+   </listitem>
+   <listitem>
+    <para>Virtual guest nodes running &pmremote;. In this guide, one guest
+     node is named <systemitem class="domainname">&wsII;</systemitem>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     &pace; to manage guest nodes and remote nodes.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.term">
+  <title>Terminology</title>
+  <variablelist>
+   <varlistentry>
+    <term>Pacemaker Remote (&pmremote;)</term>
+    <listitem>
+     <para>
+      A service daemon that makes it possible to use a node as a &pace;
+      node without deploying the full cluster stack. While nodes that run
+      this service may run cluster resources and most
+      commandline tools, they do not support other functions of full cluster
+      nodes. This includes fencing execution, quorum voting, and DC
+      eligibility.
+     </para>
+     <note>
+      <title>Different Naming for Service and Daemon</title>
+      <para>The systemd service of Pacemaker Remote is named &pmremote;
+       whereas the name or the process/daemon is &pmrm_daemon;
+       (a trailing <quote>d</quote> after its name).
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Cluster Node</term>
+    <listitem>
+     <para>
+      A node that runs all &pace; components along with the complete
+      high-availability &corosync; stack. Cluster nodes may perform the
+      following tasks:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>Run cluster resources.</para>
+      </listitem>
+      <listitem>
+       <para>Run all command-line tools, such as <command>crm</command>,
+        <command>crm_mon</command>, and others.</para>
+      </listitem>
+      <listitem>
+       <para>Count toward cluster quorum.</para>
+      </listitem>
+      <listitem>
+       <para>Serve as the cluster's designated coordinator (DC).</para>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Remote Node</term>
+    <listitem>
+     <para>
+      A physical machine that runs the &pmremote; daemon. Remote nodes
+      includes a special resource (often called <quote>baremetal case</quote>)
+      for managing communication wtih the cluster.
```

Comment:
A special resource (ocf:pacemaker:remote) is required to be running on a corosync node for managing communication with the baremetal remote node.